curriculum/challenges/english/25-front-end-development/workshop-fcc-authors-page/641da5462576784453146ec2.md

```diff
@@ -33,22 +41,29 @@ You should use `let` to declare a variable named `endingIndex`.
 assert.match(code, /let\s+endingIndex/)
 ```
 
+Your `endingIndex` variable should be a number.
+  
+  ```js
+assert.isNumber(endingIndex, 'endingIndex should be a number')
+```
+
 You should set your `endingIndex` variable to `8`.
 
 ```js
-assert.match(code, /let\s+endingIndex\s*=\s*8\s*;?/)
+assert.strictEqual(endingIndex, 8, 'endingIndex should be set to 8')
 ```
 
-You should use `let` to declare a variable named `authorDataArr`.
+You should have a variable named `authorDataArr`.
 
 ```js
-assert.match(code, /let\s+authorDataArr/)
+assert.isDefined(authorDataArr, 'authorDataArr should be defined');
 ```
 
-You should set your `authorDataArr` variable to an empty array (`[]`).
+Your `authorDataArr` variable should be an empty array.
 
 ```js
-assert.match(code, /let\s+authorDataArr\s*=\s*\[\s*\]\s*;?/)
+assert.isArray(authorDataArr, 'authorDataArr should be an array');
+assert.deepEqual(authorDataArr, [], 'authorDataArr should be an empty array');
```

Comment:
comments needs to be removed here like the other ones.
But also for the second assert, isEmpty would be better 
https://www.chaijs.com/api/assert/#method_isempty

Reply:
thanks for the pointer.
latest commit resolves all your suggested changes